pkg/vsphere/disk/disk_manager.go

```diff
@@ -421,3 +423,66 @@ func (m *Manager) devicePathByURI(op trace.Operation, datastoreURI *object.Datas
 
 	return sysPath, nil
 }
+
+// AttachAndMount creates and attaches a vmdk as a non-persistent disk, mounts it, and returns the mount path.
```

Comment:
Just want to note here for the future. Once #5651 is merged you should be able to rebase on upstrream/master and remove this change :)